frontend_lib/src/action.async.js

```diff
@@ -76,8 +76,9 @@ export const putContentRestoreDelete = (apiUrl, workspaceId, contentId) => {
   })
 }
 
-export const getMyselfKnownMember = (apiUrl, userNameToSearch, workspaceIdToInclude) => {
-  return fetch(`${apiUrl}/users/me/known_members?acp=${userNameToSearch}&include_workspace_ids=${workspaceIdToInclude}`, {
+export const getMyselfKnownMember = (apiUrl, userNameToSearch, workspaceIdToInclude, limit = 0) => {
+  const limitParam = limit ? '&limit=' + limit : ''
```

Comment:
```suggestion
  const limitParam = limit ? `&limit=${limit}` : ''
```

Reply:
Not a big fan of this, but committing your suggestion anyway (no strong opinion).